crates/red_knot_module_resolver/build.rs

```diff
@@ -22,9 +22,15 @@ const TYPESHED_ZIP_LOCATION: &str = "/zipped_typeshed.zip";
 /// <https://github.com/zip-rs/zip-old/blob/5d0f198124946b7be4e5969719a7f29f363118cd/examples/write_dir.rs>
 fn zip_dir(directory_path: &str, writer: File) -> ZipResult<File> {
     let mut zip = ZipWriter::new(writer);
+    #[cfg(not(target_arch = "wasm32"))]
```

Comment:
How does this work when compiling under wasm? Doesn't this mean the following statement will be eliminated under wasm, which would just mean the variable `method` becomes undefined?

Reply:
Ah yeah. I should remove this. My hope was that I could use regular `cfg` attributes in the build script but this doesn't work because the `target_arch` in the build script is the system's `arch` running the build script.